go.mod

```diff
@@ -67,6 +67,7 @@ require (
 	github.com/hdevalence/ed25519consensus v0.2.0 // indirect
 	github.com/josharian/native v1.1.1-0.20230202152459-5c7d0dd6ab86 // indirect
 	github.com/jsimonetti/rtnetlink v1.4.0 // indirect
+	github.com/kylelemons/godebug v1.1.0 // indirect
```

Comment:
Is this used?

Reply:
Added automatically by `go mod tidy`

Reply:
Ok this comes from the prometheus update to >1.20:  https://github.com/cloudoperators/greenhouse/pull/447, which introduces the dependency here:
https://github.com/prometheus/client_golang/commit/bccd68204bf41e9cce27e8561b70e0d1cb350bcb